docs/source/examples/mitiq-paper/mitiq-paper-codeblocks.md

```diff
@@ -167,35 +167,29 @@ print("Original circuit:", circ, sep="\n")
 +++ {"id": "_5Q3P1DJfcar"}
 
 **Codeblock 9**
+<!-- #endregion -->
 
-```{code-cell} ipython3
----
-colab:
-  base_uri: https://localhost:8080/
-id: tSWVDcJIeuck
-outputId: e9d0c874-8814-4da0-c54b-a6881d3e614a
----
-folded = zne.scaling.fold_gates_from_left(
-    circ, scale_factor=2
-)
-print("Folded circuit:", folded, sep="\n")
+> Note: The method `zne.scaling.fold_gates_from_left` has been deprecated as of v0.36.0
+
+```python colab={"base_uri": "https://localhost:8080/"} id="tSWVDcJIeuck" outputId="e9d0c874-8814-4da0-c54b-a6881d3e614a"
+# folded = zne.scaling.fold_gates_from_left(
+#     circ, scale_factor=2
+# )
+# print("Folded circuit:", folded, sep="\n")
 ```
 
 +++ {"id": "4ocoSlNIfnV0"}
 
 **Codeblock 10**
+<!-- #endregion -->
```

Comment:
Any idea what this is from?

Reply:
Nope, but I am guessing it's some artifact from the conversion between Myst and Jupyter formats mentioned [here](https://github.com/unitaryfund/mitiq/pull/2289#discussion_r1567674306).

Reply:
Is it possible it's something from you editor relating to code folding? There's an example of this syntax [here](https://marketplace.visualstudio.com/items?itemName=maptz.regionfolder).

Reply:
Possible, though I haven't used that workflow before in VSCode. I removed in the .md.